Info.lua

```diff
@@ -6,7 +6,7 @@
 g_PluginInfo = 
 {
 	Name = "WorldEdit",
-	Version = 2,
+	Version = 1, -- Set to 1 to test the updater
```

Comment:
I just uploaded this code so it can be tested or to see if [this](http://forum.mc-server.org/showthread.php?tid=870&pid=24669#pid24669) is actually the cause. When all this code works I'll squash everything in one commit.